Emby.Server.Implementations/Library/Resolvers/Audio/MusicArtistResolver.cs

```diff
@@ -15,20 +15,20 @@ namespace Emby.Server.Implementations.Library.Resolvers.Audio
     /// </summary>
     public class MusicArtistResolver : ItemResolver<MusicArtist>
     {
-        private readonly ILogger _logger;
+        private readonly ILogger<MusicAlbumResolver> _logger;
```

Comment:
The class is MusicArtist so why use the Album one?

Reply:
That instance is passed through to created instances of `MusicAlbumResolver` and never used by the `MusicArtistResolver` class itself